src/main/java/com/superdevs/HealthOMeter/dto/AuthorityDto.java

```diff
@@ -0,0 +1,31 @@
+package com.superdevs.HealthOMeter.dto;
+
+public class AuthorityDto {
+
+    private Long id;
```

Comment:
tak samo jak wyzej

Reply:
Fixing it right now, great suggestion, make sense.